events/managers.py

```diff
@@ -172,6 +172,9 @@ def buy_a_bet(self, user, event_id, bet_outcome, price):
 
         user, event, bet = self.get_user_event_and_bet_for_update(user, event_id, bet_outcome)
 
+        if event.to_be_resolved:
+            raise EventWaitingToBeResolved(_('Event waiting to be resolved'))
```

Comment:
please manage this and the one raised in sell_a_bet in `events.views.create_transaction`, where buy and sell methods are called